test/models/service_test.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path')
+const assert = require('assert')
+const Service = require(path.join(__dirname, '/../../app/models/Service.class'))
+
+describe('Service model from service raw data', function () {
+  it('should contain expected merchant details country name', function () {
+    let data = {
+      external_id: '1234',
+      name: 'service name',
+      gateway_account_ids: [1],
+      custom_branding: {css_url: 'css url', image_url: 'image url'},
+      merchant_details: {
+        name: 'Give Me Your Money',
+        address_line1: 'Clive House',
+        address_line2: '10 Downing Street',
+        address_city: 'London',
+        address_postcode: 'AW1H 9UX',
+        address_country: 'GB'
+      }
+    }
+
+    let serviceModel = new Service(data)
+
+    assert.equal(serviceModel.merchantDetails.countryName, 'United Kingdom')
```

Comment:
think we're trying to standardise on `expect` no? Check with @maxcbc

Reply:
Yeah. We've been trying to standardise on chai's expect style. Currently we're using at least 3 different styles across the projects.

Happy to help out if you're unfamiliar with `expect`.